lib/perl/Genome/Model/Tools/Analysis/LaneQc/CompareSnpsBuildLanes.pm

```diff
@@ -143,8 +144,8 @@ sub execute {
 samtools pileup -vc -f $reference_file $alignment_file | perl -pe '\@F = split /\\t/; \\\$_=q{} unless(\\\$F[7] > 2);' > $dir/$lane_name.var
 gmt analysis lane-qc compare-snps --genotype-file $genotype_file --variant-file $dir/$lane_name.var > $dir/$lane_name.var.compare_snps
 COMMANDS
-                my $email_domain = Genome::Config::get('email_domain');
-                print `bsub -N -u $user\@$email_domain "$command"`;
+                my $email = $user->email;
+                print `bsub -N -u $email "$command"`;
```

Comment:
Yeah... this code is old and busted, to be sure!  I'm reluctant to put too much time into fixing it until it survives <https://jira.gsc.wustl.edu/browse/APIPE-3367>.  If we did change it, I might use `Genome::Sys::LSF::bsub` (which itself eventually uses `IPC::System::Simple`).

A lot of the commands under `Genome::Model::Tools` are user-contributed and have been left relatively untouched (save for little fixes like this).  This one is from 2010 and was written by someone who left a few years ago; I'll be curious to check the usage log to see if anyone has still been running it.  (As a side note, the git history is redacted before 2012 on this public repository to remove anything we lack permission to share, but the older history is available from the internal server.)